test/kotlin/apps/baseline/MainActivity.kt

```diff
@@ -128,7 +126,7 @@ class MainActivity : Activity() {
           Log.d("MainActivity", "filter-demo: $filterDemoValue")
         }
 
-        if (status == 200) {
+        if (status != 100) {
```

Comment:
ah, I see - we want to show 301 requests as successes in the UI even though it does not matter from the perspective of the CI job itself - makes sense, that will make it easier for us by human beings who want to run these apps manually. 👍

In that case, can we have a check that looks as follows `status >= 200 && status <= 301` + add some comment on top of this code explaining that it's expected that `http://api.lyft.com/ping` returns us a 301? Just to make this code easier to understand for the potential reader (and future us 🙃)

Reply:
given only 301 will work I'm going to update to that, and comment why.